server/statistics/store.go

```diff
@@ -24,6 +24,15 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	storeStatsRollingWindowsSize = 3
+
+	// RegionsStatsObserveInterval is the interval for obtaining statistics from RegionTree
+	RegionsStatsObserveInterval = 30 * time.Second
+	// RegionsStatsRollingWindowsSize is default size of median filter for data from regionStats
+	RegionsStatsRollingWindowsSize = 9
```

Comment:
Why use 9? is it too long?

Reply:
`DefaultWriteMfSize` is 5. TiFlash needs more stable statistics. After testing, the output is more stable when it is 9.